src/v2/components/Transactions/index.jsx

```diff
@@ -16,33 +14,12 @@ import useStyles from './styles';
 
 const TransactionsPage = ({match}: {match: Match}) => {
   const classes = useStyles();
-  const {
-    transactions,
-    transactionCount,
-    start,
-    next,
-    prev,
-  } = TransactionsTimelineStore;
+  const {transactions, transactionCount, start} = TransactionsTimelineStore;
 
   if (start !== match.params.start) {
     TransactionsTimelineStore.init({start: match.params.start});
   }
 
-  const nav = (
```

Comment:
can we agree, please don't remove nav until we have a replacement? It's useful for debugging pagination

Reply:
pagination is still there but provided by material table, so we don't need our own buttons